src/integrationTest/java/org/opensearch/security/ConfigurationFiles.java

```diff
@@ -33,10 +33,12 @@ public static Path createConfigurationDirectory() {
                 "action_groups.yml",
                 "config.yml",
                 "internal_users.yml",
+                "nodes_dn.yml",
                 "roles.yml",
                 "roles_mapping.yml",
                 "security_tenants.yml",
-                "tenants.yml" };
+                "tenants.yml",
```

Comment:
Yes, this change is necessary so that the SecurityConfigurationBootstrapTests setup can happen. All of these config files need to be preset in order for the default security init to succeed.